src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -1058,23 +1060,58 @@ private void insureProfileDirectory(Profile profile) {
             new File(profileDirectory).mkdirs();
         }
     }
+    
+    /**
+     * Load judgements from reject.ini
+     * @param contest
+     * @param cdpConfigDirectory
+     * @return
+     * @throws Exception
+     */
+    protected boolean loadJudgements(IInternalContest contest, File cdpConfigDirectory) throws Exception {
 
+        String filename = getCdpConfigRejectIniFilename(cdpConfigDirectory);
+
+        if (new File(filename).exists()) {
+            return new JudgementLoader().loadedJudgementsFromIni(contest, filename);
+        }
+        return false;
+    }
+
+    private String getCdpConfigRejectIniFilename(File cdpConfigDirectory) {
+        return cdpConfigDirectory.getAbsolutePath() + File.separator + Constants.JUDGEMENT_INIT_FILENAME;
+    }
+
+    /**
+     * Load judgements from reject.ini if it exists.
+     */
     protected void loadJudgements() {
 
         if (!isContactingRemoteServer()) {
 
             if (contest.getJudgements().length == 0) {
-
                 if (loadedJudgementsFromIni(Constants.JUDGEMENT_INIT_FILENAME)) {
                     info("Loaded judgements from " + Constants.JUDGEMENT_INIT_FILENAME);
-                } else {
-                    info(Constants.JUDGEMENT_INIT_FILENAME + " not found.  Loading default judgements");
-                    loadDefaultJudgements();
```

Comment:
Again, unless I'm misunderstanding the PR review process/procedure/mechanisms, this looks like it is a logic change (removal of the call to `loadDefaultJudgements()` under the `else` condition), which does not seem to me to have anything to do with the purpose of Issue #119.  This should be submitted under a separate issue/PR.

Reply:
I did not make this change.